ai/mcp-server.md

```diff
@@ -44,7 +44,7 @@ You also need to add your [Telerik licence key](slug:installation-license-key) a
 
 For detailed instructions, refer to [Use MCP servers in Visual Studio](https://learn.microsoft.com/en-us/visualstudio/ide/mcp-servers).
 
-> Visual Studio 17.14 seems to require the Copilot Chat window to be open and active when you open a solution. Otherwise the Telerik MCP server is not used.
+> Older Visual Studio versions seem to require the Copilot Chat window to be open and active when you open a solution. Otherwise the Telerik MCP server is not used.
```

Comment:
Older is way too generic. If you can, be more specific - e.g., version XXX and earlier.

Reply:
I am not sure about the exact number, because it's a sub version of 17.14.